index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <!-- ________ *ALCUNOO* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">ALCUNO</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank"> </a>
```

Comment:
If you have twitter account then please put it here, else you can also put your github Id and change the "href" and "i tag" according to it.

Reply:
I don't have a Twitter, I'll probably add the GitHub ID.